src/kibana/plugins/vis_types/vislib/editors/tile_map.html

```diff
@@ -1,19 +1,122 @@
+    >
   </select>
 </div>
 
-<div class="vis-option-item">
-  </br>
+<div ng-if="vis.params.mapType === 'Heatmap'" class="form-group">
+  <div>
+    <label>
+      Radius
+      &nbsp;<kbn-info placement="right" info="Size of heatmap dots. Default: 25"></kbn-info>
+    </label>
+    <div class="vis-editor-agg-form-row">
+      <input
+        name="heatRadius"
+        ng-model="vis.params.heatRadius"
+        required
+        class="form-control"
+        type="range"
+        min="5"
+        max="50"
+        step="1"
+        >
+      <div class="form-group vis-editor-agg-form-value">
+        {{vis.params.heatRadius}}
+      </div>
+    </div>
+  </div>
+  <div>
+    <label>
+      Blur
+      &nbsp;<kbn-info placement="right" info="Amount of blur applied to dots. Default: 15"></kbn-info>
+    </label>
+    <div class="vis-editor-agg-form-row">
+      <input
+        name="heatBlur"
+        ng-model="vis.params.heatBlur"
+        required
+        class="form-control"
+        type="range"
+        min="1"
+        max="25"
+        step="1"
+        >
+      <div class="form-group vis-editor-agg-form-value">
+        {{vis.params.heatBlur}}
+      </div>
+    </div>
+  </div>
+  <div>
+    <label>
+      Maximum zoom
+      &nbsp;<kbn-info placement="right" info="Map zoom at which all dots are display at full intensity. Default: 16"></kbn-info>
+    </label>
+    <div class="vis-editor-agg-form-row">
+      <input
+        name="heatMaxZoom"
+        ng-model="vis.params.heatMaxZoom"
+        required
+        class="form-control"
+        type="range"
+        min="1"
+        max="18"
+        step="1"
+        >
+      <div class="vis-editor-agg-form-value">
+        {{vis.params.heatMaxZoom}}
+      </div>
+    </div>
+  </div>
+  <div>
+    <label>
+      Minimum opacity
+      &nbsp;<kbn-info placement="right" info="Minimum opacity of dots. Default: 0.1"></kbn-info>
+    </label>
+    <div class="vis-editor-agg-form-row">
+      <input
+        name="heatMinOpacity"
+        ng-model="vis.params.heatMinOpacity"
+        required
+        class="form-control"
+        type="range"
+        min="0"
+        max="1.0"
+        step="0.01"
+        >
+      <div class="vis-editor-agg-form-value">
+        {{vis.params.heatMinOpacity}}
+      </div>
+    </div>
+  </div>
+  <div class="form-group">
+      <span class="hintbox-label" ng-click="heatNormalizeData = !heatNormalizeData">
+        <label>
+          <input type="checkbox" value="{{heatNormalizeData}}" ng-model="vis.params.heatNormalizeData" name="heatNormalizeData" ng-checked="vis.params.heatNormalizeData">
+          Normalize data for heatmap intesity
+        </label>
+        <i class="fa fa-info-circle"></i>
+      </span>
+      <div class="hintbox" ng-show="heatNormalizeData">Default is checked to show heatmap intensity of bucket count as a percent of max count. Uncheck to show heatmap intensity simply as bucket count.</div>
```

Comment:
Should this say **bucket count**? What if they're using an avg/min/max agg? Should probably say "simply as metric value"